operators/generate-kas-fleetshard-olm-bundle.sh

```diff
@@ -180,13 +180,14 @@ initialize_inputs() {
     rm -rf ${CRD_DIR} && mkdir ${CRD_DIR}
 
     # Extract Operator cluster role (from source) and deployment (from generated output) to input directory
-    ${YQ} e '. | select(.kind == "ClusterRole")' ${OPERATOR_YAML} > ${CRD_DIR}/operator-clusterrole.yml
-    ${YQ} e '. | select(.kind == "Deployment")'  ${OPERATOR_YAML} > ${CRD_DIR}/operator-deployment.yml
+    ${YQ} e '. | select(.kind == "ClusterRole")'   ${OPERATOR_YAML} > ${CRD_DIR}/operator-clusterrole.yml
+    ${YQ} e '. | select(.kind == "Deployment")'    ${OPERATOR_YAML} > ${CRD_DIR}/operator-deployment.yml
+    ${YQ} e '. | select(.kind == "PriorityClass")' ${OPERATOR_YAML} > ${CRD_DIR}/operator-reservation-priorityclass.yml
```

Comment:
Should the name here be just operator-priorityclass?  @rareddy had wanted to move the other priority class into the kubernetes.yml, and there may be more down the road (such as giving zookeeper and broker an elevated priority wrt canary and admin)

Reply:
The file name at this step of the script is arbitrary. This will need some updates to support more than the single PriorityClass. If we're ready for it now, I'll make the change.

Reply:
Added support for multiple PriorityClass resources - keeping the included `kas-fleetshard-high` class for now.